src/views/AnnualReport.vue

```diff
@@ -96,13 +108,12 @@
                     <p>Verify or change your Registered Office Addresses.</p>
                   </header>
                   <office-addresses
-                    :addresses.sync="addresses"
-                    :registeredAddress.sync="registeredAddress"
-                    :recordsAddress.sync="recordsAddress"
-                    :asOfDate="asOfDate"
+                    ref="officeAddressesComponent"
+                    :addresses.sync="updatedAddresses"
                     :componentEnabled="allowChange('coa')"
+                    @original="originalAddresses=$event"
                     @modified="officeModifiedEventHandler($event)"
-                    @valid="addressFormValid = $event"
+                    @valid="addressesFormValid=$event"
```

Comment:
Note the new interface (API) between the parent and child.
In addition, the parent makes a call to one of the child's methods to load data. I don't like this but it's the only way to make the fetch async.
A better architecture would be for the parent to do the fetch (or put it in a common mixin/service). That will be another day.